### DISCUSSION TITLE
Postgresql time stamp with time zone conversion failed

### DISCUSSION THREAD

**Discussion by AliMacky** [prisma/prisma#25924] [Q&A]
### Question

Hello, I've been trying to get prisma to work with a created_at and updated_at column in my database, but im getting this error when I try to query to/from the DB: 
```
prisma:error 
Invalid `prisma.admin_users.findFirst()` invocation:


Inconsistent column data: Conversion failed: expected a datetime string in column 'created_at', found 2024-12-24 08:25:49.625229-08
```

heres my prisma schema:
```
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["driverAdapters"]
  binaryTargets = ["native", "linux-arm64-openssl-3.0.x", "debian-openssl-3.0.x"]
}
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model admin_users {
  id                 Int                  @id @default(autoincrement())
  username           String               @db.VarChar(100)
  email              String               @unique @db.VarChar(100)
  password           String               @db.VarChar(100)
  created_at         DateTime?            @default(now()) @db.Timestamptz(6)
  updated_at         DateTime?            @default(now()) @db.Timestamptz(6)
  updated_by         Int?
  admin_users        admin_users?         @relation("admin_usersToadmin_users", fields: [updated_by], references: [id], onUpdate: NoAction)
  other_admin_users  admin_users[]        @relation("admin_usersToadmin_users")
  user_accounts      user_accounts[]
  moderation         moderation[]
  moderation_archive moderation_archive[]
  sticky_posts       sticky_posts[]
  topics             topics[]
}
```

**EDIT:**
and heres the schema for my postgresql table:
```
CREATE TABLE IF NOT EXISTS admin_users (
    id SERIAL PRIMARY KEY NOT NULL,
    username VARCHAR(100) NOT NULL,
    email VARCHAR(100) UNIQUE NOT NULL,
    password VARCHAR(100) NOT NULL,
    created_at TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
    updated_at TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
    updated_by INTEGER REFERENCES admin_users(id)
    ON DELETE SET NULL
);
```

Any ideas why this is happening? It's working on some machines, and failing on others (with the same data in the DB).

**Comment by nurul3101** [comment]
Hi @AliMacky 👋 

> It's working on some machines, and failing on others (with the same data in the DB).

Can you elaborate on what you meant by it is working on some machines? Do you mean this query: `prisma.admin_users.findFirst()` works on few servers and doesn't work on others?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
